### DISCUSSION TITLE
Stock Recommender System PyTorch Wave App

### DISCUSSION THREAD

**Discussion by james94** [h2oai/wave#2200] [Ideas]
Starting the discussion thread for a stock recommender system powered by PyTorch and H2O Wave. If I have time, I will also add Apache NiFi's custom python extensibiliy for handling data preprocessing for stock data. I am planning to design and develop a personalized stock recommendation system. My current domain of companies relates to medical AI, but this app could be re-tuned for autonomous vehicles, unmanned aircraft systems, fintech and other companies. I will keep you guys posted.

Here is the repo where the progress on this project idea resides: https://github.com/Financial-AI/Stock-Recommender-System

**Comment by mturoci** [comment]
Good luck @james94!

**Reply by james94** [reply]
Thanks @mturoci 

I will keep you guys posted.
